mubench.pipeline/tests/data/stub_detector.py

```diff
@@ -3,4 +3,5 @@
 
 class StubDetector(Detector):
     def __init__(self):
-        super().__init__("-detectors-", "StubDetector", [], cli_version=RunnerInterfaceTestImpl.TEST_VERSION)
+        self._get_release_info = lambda *_: {'tag': 'latest', 'md5': None, 'cli_version': RunnerInterfaceTestImpl.TEST_VERSION}
```

Comment:
The question is, should the info be loaded by the constructor? Can we load it at a later point to avoid this race condition?